vm-setup/roles/v1a2_integration_test/templates/cluster.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: cluster.x-k8s.io/v1alpha2
+kind: Cluster
+metadata:
+  name: {{ CLUSTER_NAME }}
+spec:
+  clusterNetwork:
+    services:
+      cidrBlocks: ["10.96.0.0/12"]
+    pods:
+      cidrBlocks: ["192.168.0.0/18"]
+    serviceDomain: "cluster.local"
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+    kind: BareMetalCluster
+    name: {{ CLUSTER_NAME }}
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+kind: BareMetalCluster
+metadata:
+  name: {{ CLUSTER_NAME }}
+spec:
+ apiEndpoint: http://192.168.111.249:6443
```

Comment:
This should be derived from baremetal_network_cidr not hardcoded

Reply:
This could be done later, along doing it in the baremetal cluster CR